scripts/seed.ts

```diff
@@ -0,0 +1,114 @@
+import { config } from 'dotenv';
+import pg from 'pg';
+
+config();
+
+async function seed() {
+  const client = new pg.Client({
+    user: process.env.POSTGRES_USER,
+    password: process.env.POSTGRES_PASSWORD,
+    host: process.env.POSTGRES_HOST,
+    port: parseInt(process.env.POSTGRES_PORT || '5432'),
+    database: process.env.POSTGRES_DB,
+    ssl: process.env.POSTGRES_SSL === '1',
+  });
+
+  try {
+    await client.connect();
+
+    // Create admin business entity
+    const adminEntityResult = await client.query(`
+      INSERT INTO accounter_schema.financial_entities (type, name)
+      VALUES ('business', 'Admin Business')
```

Comment:
Maybe we can use `inquirer` tool (used [here](https://github.com/Urigo/accounter-fullstack/blob/main/packages/modern-poalim-scraper/src/scrapers/hapoalim.ts#L46-L52) for example) to enable CLI control over the admin business name

Reply:
The main goal was to make it easy to setup. So basically any dev can clone the repo and get going without having to really think.

With inquirer you need the dev to go answer questions. And could be a good feature in the future, but was less what I was trying to accomplish here.

Also, if the goal were for this to be user focused I'd suggest having a form in the UI, rather than building out CLI's that only technical users can use.

Reply:
Also if a CI/CD system were to run this you don't want it running inquirer.

You'll support for seeding in libraries like Prisma. We're basically doing the same thing in this PR.
https://www.prisma.io/docs/orm/prisma-migrate/workflows/seeding